<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>675</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Drachenflug-Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="drachenListWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_drachenDetails">
        <property name="text">
         <string>Details zum Drachen:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="detailTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_Plaene">
        <property name="text">
         <string>Reisepläne</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="reisePlanListWidget"/>
      </item>
      <item>
       <widget class="QLabel" name="label_reisen">
        <property name="text">
         <string>Drachenreisen:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="reiseListWidget"/>
      </item>
      <item>
       <widget class="QLabel" name="label_drachenflugDetails">
        <property name="text">
         <string>Details zur Drachenreise:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="reiseDetailsListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="addDracheButton">
          <property name="text">
           <string>Drache hinzufügen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addReiseplanButton">
          <property name="text">
           <string>Reiseplan hinzufügen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addDrachenreiseButton">
          <property name="text">
           <string>Reise hinzufügen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addPassengerButton">
          <property name="text">
           <string>Passagier buchen</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>Speichern</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>18</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionEinlesen"/>
    <addaction name="actionSpeichern"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuL_schen">
    <property name="title">
     <string>Löschen</string>
    </property>
    <addaction name="actionDrache_L_schen"/>
    <addaction name="actionReiseplan_L_schen"/>
    <addaction name="actionDrachenreise_L_schen"/>
    <addaction name="actionPassagier_L_schen"/>
   </widget>
   <widget class="QMenu" name="menuSuchen">
    <property name="title">
     <string>Suchen</string>
    </property>
    <addaction name="actionPassagierSuchen"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuL_schen"/>
   <addaction name="menuSuchen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBeenden"/>
   <addaction name="actionSpeichern"/>
   <addaction name="actionEinlesen"/>
   <addaction name="actionDrache_L_schen"/>
   <addaction name="actionReiseplan_L_schen"/>
   <addaction name="actionDrachenreise_L_schen"/>
   <addaction name="actionPassagier_L_schen"/>
   <addaction name="actionPassagierSuchen"/>
  </widget>
  <action name="actionBeenden">
   <property name="icon">
    <iconset>
     <normaloff>icons/beenden.svg</normaloff>icons/beenden.svg</iconset>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
    </font>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset>
     <normaloff>icons/speichern.svg</normaloff>icons/speichern.svg</iconset>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
  <action name="actionEinlesen">
   <property name="icon">
    <iconset>
     <normaloff>icons/einlesen.svg</normaloff>icons/einlesen.svg</iconset>
   </property>
   <property name="text">
    <string>Einlesen</string>
   </property>
  </action>
  <action name="actionDrache_L_schen">
   <property name="icon">
    <iconset>
     <normaloff>icons/drache.svg</normaloff>icons/drache.svg</iconset>
   </property>
   <property name="text">
    <string>Drache Löschen</string>
   </property>
  </action>
  <action name="actionReiseplan_L_schen">
   <property name="icon">
    <iconset>
     <normaloff>icons/reiseplan.svg</normaloff>icons/reiseplan.svg</iconset>
   </property>
   <property name="text">
    <string>Reiseplan Löschen</string>
   </property>
  </action>
  <action name="actionDrachenreise_L_schen">
   <property name="icon">
    <iconset>
     <normaloff>icons/drachenreise.svg</normaloff>icons/drachenreise.svg</iconset>
   </property>
   <property name="text">
    <string>Drachenreise Löschen</string>
   </property>
  </action>
  <action name="actionPassagier_L_schen">
   <property name="icon">
    <iconset>
     <normaloff>icons/passagier.svg</normaloff>icons/passagier.svg</iconset>
   </property>
   <property name="text">
    <string>Passagier Löschen</string>
   </property>
  </action>
  <action name="actionPassagierSuchen">
   <property name="icon">
    <iconset>
     <normaloff>icons/suchen.svg</normaloff>icons/suchen.svg</iconset>
   </property>
   <property name="text">
    <string>Passagier Suchen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
